primer.html

```diff
@@ -448,7 +450,7 @@ <h2>Creating a New Container for ShapeTrees</h2>
     </section>
 
     <section id="updating">
-      <h2>Updating ShapeTree Instances <a class="cross" href="spec#post-managed"></a></h2>
+      <h2>Updating ShapeTree Instances <a class="cross" href="spec.html#post-managed"></a></h2>
```

Comment:
(plus three more places below)

I typically avoid extensions because caching, location history, and metadata assertions work better if there's only one form a a URL. That said, I recognize that filesystems don't provide conneg and running an http server may be a burden during spec dev.

Reply:
Sorry, didn't realize this was working correctly in Jekyll.  Reverted this change.